cob_bringup/robots/cob3-2.xml

```diff
@@ -42,15 +42,15 @@
 		<!-- lower robot -->
 		<include file="$(find cob_bringup)/drivers/sick_s300.launch" >
 			<arg name="robot" value="$(arg robot)" />
-			<arg name="name" value="front" />
+			<arg name="name" value="base_laser_front" />
```

Comment:
what do you think about namespacing like we have done it with the controllers. Eg: `base/laser/front`